ui_framework/src/components/table/listing_table/listing_table_toolbar_footer.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  KuiToolBarFooter,
+  KuiToolBarText,
+  KuiToolBarFooterSection,
+} from '../../';
+
+export function KuiListingTableToolbarFooter({ pagerComponent, itemsSelectedCount }) {
+  return (
+    <KuiToolBarFooter>
+      <KuiToolBarFooterSection>
+        <KuiToolBarText>
+          {itemsSelectedCount > 0 && `${itemsSelectedCount} items selected`}
```

Comment:
Can we update this logic so that if `itemsSelectedCount` is 1 then the noun is singular ("item")?